README.md

```diff
@@ -31,82 +69,134 @@ Add this plugin like this:
+### `sbot.metafeeds.findOrCreate(metafeed, find, details, cb)`
 
-## API
+Looks for the first subfeed of `metafeed` that satisfies the condition in `find`,
+or creates it matching the properties in `details`.
+
+This is strictly concerned with meta feeds and sub feeds that **you own**, not
+with those that belong to other peers.
+
+Arguments:
+- `metafeed` - the metafeed you are finding/ creating under, can be:
+    - *FeedDetail* object (as returned by `findOrCreate()` or `getRoot()`)
+    - *null* which is short-hand for the `rootFeed` (this will be created if doesn't exist)
+- `find` - method you use to find an existing *FeedDetail*, can be:
+    - *function* of shape `(FeedDetail) => boolean`
+    - *null* - this method will then return an arbitrary subfeed under provided `metafeed`
+- `details` - used to create a new subfeed if a match for an existing one is not found, can be 
+    - *Object*: 
+        - `details.feedpurpose` *String* any string to characterize the purpose of this new subfeed
+        - `details.feedformat` *String* either `'classic'` or `'bendybutt-v1'`
+        - `details.metadata` *Object* (optional) - for containing other data
+            - if `details.metadata.recps` is used, the subfeed announcement will be encrypted
+    - *null* - only allowed if `metafeed` is null (i.e. the details of the `root` FeedDetail)
+- `cb` *function* delivers the response, has signature `(err, FeedDetail)`, where FeedDetail is
+    ```js
+    {
+      metafeed: 'ssb:feed/bendybutt-v1/sxK3OnHxdo7yGZ-28HrgpVq8nRBFaOCEGjRE4nB7CO8=',
+      // metafeed = the feed this is a subfeed of
+      feedpurpose: 'chess',
+      feedformat: 'classic',
+      id: '@I5TBH6BuCvMkSAWJXKwa2FEd8y/fUafkQ1z19PyXzbE=.ed25519',
+      seed: <Buffer 13 10 25 ab e3 37 20 57 19 0a 1d e4 64 13 e7 38 d2 23 11 48 7d 13 e6 3b 8f ef 72 92 7f db 96 64>
+      keys: {
+        curve: 'ed25519',
+        public: 'I5TBH6BuCvMkSAWJXKwa2FEd8y/fUafkQ1z19PyXzbE=.ed25519',
+        private: 'Mxa+LL16ws7HZhetR9FbsIOsAeud+ii+9KDUisXkq08jlMEfoG4K8yRIBYlcrBrYUR3zL99Rp+RDXPX0/JfNsQ==.ed25519',
+        id: '@I5TBH6BuCvMkSAWJXKwa2FEd8y/fUafkQ1z19PyXzbE=.ed25519'
+      },
+      metadata: { // example
+        notes: 'private testing of chess dev',
+        recps: ['@I5TBH6BuCvMkSAWJXKwa2FEd8y/fUafkQ1z19PyXzbE=.ed25519']
+      },
+    }
+    ```
 
 ### `sbot.metafeeds.findById(feedId, cb)`
 
 Given a `feedId` that is presumed to be a subfeed of some meta feed, this API
-fetches the metadata associated with the creation of this subfeed on its parent
-meta feed, and returns (via the callback `cb` on the second argument) an object
-with the following shape:
+fetches the *Details* object describing that feed, which is of form:
 
+```js
+{
+  metafeed,
+  feedpurpose,
+  feedformat,
+  id,
+  // seed
+  // keys
+  metadata
+}
 ```
-{ feedpurpose, feedformat, metafeed, metadata }
-```
+
+NOTE - may include `seed`, `keys` if this is one of your feeds.
```

Comment:
:fire: is this true?
This current signature really suggests you can use this on other peoples feedIds too....